operations/host_spawn.go

```diff
@@ -641,3 +641,44 @@ func hostTerminate() cli.Command {
 		},
 	}
 }
+
+func hostRunCommand() cli.Command {
+	const commandFlagName = "command"
+
+	return cli.Command{
+		Name:  "run-command",
+		Usage: "run a bash shell script on the host",
+		Flags: addHostFlag(
+			cli.StringFlag{
+				Name:  commandFlagName,
+				Usage: "script to pass to bash",
+			}),
+		Before: mergeBeforeFuncs(setPlainLogger),
+		Action: func(c *cli.Context) error {
+			confPath := c.Parent().Parent().String(confFlagName)
+			hostID := c.String(hostFlagName)
+			command := c.String(commandFlagName)
+
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			conf, err := NewClientSettings(confPath)
+			if err != nil {
+				return errors.Wrap(err, "problem loading configuration")
+			}
+			client := conf.setupRestCommunicator(ctx)
+			defer client.Close()
+
+			output, err := client.RunHostCommand(ctx, hostID, command)
+			if err != nil {
+				return errors.Wrap(err, "problem running command")
+			}
+
+			for _, line := range output {
```

Comment:
It might be clearer to the user if we somehow make it obvious that this is writing the output from the host running the script (maybe put something like `grip.Info("Output:")` before this).

Reply:
I had thought about doing this, but thought that if we intend to use it programmatically (such as in a VSCode extension) it would be easier if there wasn't anything extra in the output.
If you feel strongly I could be persuaded otherwise, though.

Reply:
I also thought about this and that's fair. Can you add to the Usage description that it outputs the script output?

Reply:
done